cmds/info.c

```diff
@@ -229,6 +229,10 @@ static int fill_cmdline(void *arg)
 	for (i = 0, p = buf; i < ret; i++, p++) {
 		if (*p == '\0')
 			*p = ' ';
+		if (*p == '\n') {
+			memcpy(p, p+1, ret-i);
```

Comment:
I didn't take a look for this change yet, but put a single white space on the left and right of "+" and "-" signs.

Reply:
I modified it.